Documentation/Storage-Configuration/Object-Storage-RGW/ceph-object-swift.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Object Store with Keystone and Swift
+---
+
+You can configure the Swift API and the Keystone integration of Ceph RGW natively via the Object Store CRD, which allows native integration of the Rook operated Ceph RGW into [OpenStack](https://www.openstack.org/) clouds.
+
+## Create a Local Object Store with Keystone and Swift
+
+This example will create a `CephObjectStore` that start the RGW service in the cluster providing a Swift API.
+Using Swift requires the use of [OpenStack Keystone](https://docs.openstack.org/keystone/latest/) as an authentication provider.
+
+!!! note
+This sample requires *at least 3 bluestore OSDs*, with each OSD located on a *different node*.
```

Comment:
I think this note makes the most sense immediately before the yaml below. I was confused about what "sample" meant here until I read and re-read this a couple times.

Reply:
c6495fc050b2cf232c118ae2711e44df87a1f6b0